src/passes/CoalesceLocals.cpp

```diff
@@ -595,8 +595,14 @@ void CoalesceLocalsWithLearning::pickIndices(std::vector<Index>& indices) {
 
 // declare passes
 
-static RegisterPass<CoalesceLocals> registerPass1("coalesce-locals", "reduce # of locals by coalescing");
```

Comment:
Looks like PassRegistry is still needed, since the command line options want to map pass name strings to passes. Adjusted the registry to allow this.